src/smartdc/lib/smartos_prompt_config.sh

```diff
@@ -1255,23 +1255,24 @@ your own zpool.\n"
 
 	message="
 SmartOS can boot off either the \"zones\" zpool, or a dedicated named
-zpool in lieu of a USB stick or a CD-ROM.  Enter 'yes' if you wish to try and
-make a SmartOS zpool self-booting.\n"
+zpool in lieu of a USB stick or a CD-ROM.  Enter a pool name if you wish to
+try and make a SmartOS zpool self-booting.  Enter \"none\" to not create
+a self-booting pool.\n"
 
 	
 	if [[ $(getanswer "skip_instructions") != "true" ]]; then
 	    printf "$message"
 	    echo "Available pre-created pools: " $(zpool list -Ho name)
 	fi
 
-	promptopt "Specify a (configured) zpool from which to boot" "none" \
-		"bootpool"
+	promptopt "Specify a (configured) zpool from which to boot" \
+		${BOOTPOOL-"none"} "bootpool"
```

Comment:
I don't think this is right?

Reply:
It starts at "none".  If you have to loop back around because you messed up an answer, this will *remember* your prior answer (the way IP address, DNS parameters, etc. get remembered).

Reply:
TIL. Specifically:

https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html

When not performing substring expansion, using the form described below (e.g., ‘:-’), Bash tests for a parameter that is unset or null. Omitting the colon results in a test only for a parameter that is unset. Put another way, if the colon is included, the operator tests for both parameter’s existence and that its value is not null; if the colon is omitted, the operator tests only for existence.